projects/angular/src/popover/signpost/_signposts.clarity.scss

```diff
@@ -366,11 +366,15 @@
 
   .signpost-content-header {
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
```

Comment:
There should be a gap between the X icon and the text. As of now if you have longer text it will stick to the icon.

Reply:
added gap 4px as per figma